Calibration/HcalCalibAlgos/src/RecAnalyzerMinbias.cc

```diff
@@ -35,58 +36,64 @@ class RecAnalyzerMinbias : public edm::EDAnalyzer {
 public:
   explicit RecAnalyzerMinbias(const edm::ParameterSet&);
   ~RecAnalyzerMinbias();
-  virtual void analyze(const edm::Event&, const edm::EventSetup&);
-  virtual void beginJob() ;
-  virtual void endJob() ;
+
+  virtual void analyze(const edm::Event&, const edm::EventSetup&) override;
```

Comment:
@Dr15Jones I see that here and in many other places plain `TFile`s are used as output.
A possible improvement would be to move all of these to use `TFileService`.